pub_sub/go/sdk/order-processor/app.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/dapr/go-sdk/service/common"
+	daprd "github.com/dapr/go-sdk/service/http"
+)
+
+var sub = &common.Subscription{
+	PubsubName: "order_pub_sub",
+	Topic:      "orders",
+	Route:      "orders",
+}
+
+func main() {
+	s := daprd.NewService(":6001")
+	http.HandleFunc("/orders", handleRequest)
```

Comment:
Just a general comment on naming conventions. I think we should try to keep naming more uniform across different language examples for example subscribe, publish apis should be named that way across different examples. Just to keep the experience seamless even when languages change.